playbooks/hc-ansible-deployment/node_replace_inventory.yml

```diff
@@ -0,0 +1,10 @@
+cluster_node:
+  hosts:
+   # mention whatever host is provided in gluster_maintenance_cluster_node or in gluster_maintenance_cluster_node_2 either one.
```

Comment:
Improve the comment section. What should be the hostname that should here should be clear. There shouldn't be ambiguity. Something like : "this is the backend network FQDN of one of the active hosts. It can take a value of the hosts in gluster_maintenance_cluster_node
1. This <host2-backend-network-FQDN> can be better named as <active-host-backend-network-FQDN>